### ISSUE TITLE
Async001...Async006 is not picked up in Microsoft.CodeQuality.CSharp.Analyzers

### ISSUE THREAD

**Issue by subhendu-de** [dotnet/roslyn-analyzers#1954]
I am using Microsoft.CodeQuality.CSharp.Analyzers and following rules are set to Error in the ruleset file.
- [ ] Async001: Avoid Async Void
- [ ] Async002: Async Method Names Should End in Async
- [ ] Async003: Don't Pass Async Lambdas as Void Returning Delegate Types
- [ ] Async004: Don't Store Async Lambdas as Void Returning Delegate Types
- [ ] Async005: Propagate CancellationTokens When Possible
- [ ] Async006: Don't Mix Blocking and Async

But none of these rules are picked up while building the solution. I am using following configuration
- [ ] Microsoft.CodeAnalysis.FxCopAnalyzers version 2.6.2
- [ ] Visual Studio Enterprose 2017 version 15.8.9
- [ ] .NET Standard Library 2.0

Following lines were added to configure the ruleset and stylecop.json
```xml
  <PropertyGroup>
    <CodeAnalysisRuleSet>..\..\..\References\CodeAnalysis\Sample.ruleset</CodeAnalysisRuleSet>
  </PropertyGroup>
  <ItemGroup>
    <AdditionalFiles Include="..\..\..\References\StyleCop\stylecop.json" />
  </ItemGroup>
```

Any pointers why do they not picked up?

**Comment by subhendu-de** [comment]
Thanks @mavasani for the information. I have found following two static analyzers regarding Async analyzers
https://www.nuget.org/packages/microsoft.visualstudio.threading.analyzers
https://www.nuget.org/packages/AsyncUsageAnalyzers
@sharwell any guidance which one to choose? The first one is quite extensive than second one.

**Comment by mavasani** [comment]
Also tagging @AArnott

**Comment by subhendu-de** [comment]
Thanks @AArnott for the helpful information.
